packages/app/src/app/components/CodeEditor/Monaco/index.js

```diff
@@ -199,6 +202,18 @@ class MonacoEditor extends React.Component<Props, State> implements Editor {
     }
   }
 
+  setupTranspilationListener() {
+    listen(({ type, code, path }) => {
```

Comment:
This is reaaaally nice! I love that we now have the power of templates to define type definitions.